test/hardhat/e2e/constants.ts

```diff
@@ -0,0 +1,24 @@
+export const AccessController = "0xf709c8001E94e2ca6F98b7fFBCd5BD3943E46D81";
```

Comment:
Add a comment at the file level to describe what these constants are

Reply:
Also, are these addresses fixed for all deployment?

Reply:
They are fixed except the MockERC721 address, I moved it to env file